docs/source/content/getting-started.rst

```diff
@@ -33,25 +33,63 @@ Or alternately the named export in cases where it assists with integration betwe
 
 
 --------------------------------------------------
-Instantiation
+Schema
```

Comment:
that is not a schema, but an example, so it needs to be clear that this is just an example

Reply:
This should be in the example section of the instantiation, not up here

Reply:
Updated